pages/transcribe.js

```diff
@@ -203,7 +209,7 @@ const Transcribe = () => {
 
 	const formik = useFormik({
 		initialValues: {
-			sequence: ""
+			sequence: "{}"
```

Comment:
I tried this as input when trying to flag a bad image and I ran into "TypeError: a.walls is undefined" further down the line. Just a heads up.

Reply:
https://github.com/jorgev259/corridors-of-time/commit/6dbaa57d58200995fb4d242ea62b4ab43cd5653b